README.md

```diff
@@ -89,13 +89,15 @@ The following targets are supported by `nix`:
     <li>x86_64-unknown-netbsd</li>
    </td>
    <td>
+    <li>aarch64-unknown-nto-qnx710</li>
```

Comment:
You shouldn't add QNX to this table because there's no CI for it.

Reply:
Both nto items have been removed.